### DISCUSSION TITLE
autonomousSequence example is only for TWR?

### DISCUSSION THREAD

**Discussion by zul00** [bitcraze/discussions#653] [Q&A]
Hi, I was looking at the following example:

https://github.com/bitcraze/crazyflie-lib-python/blob/6370208d61cd959c1009971558b53adb43e3a24b/examples/autonomy/autonomousSequence.py#L29-L31

where it mentioned:
```
This example is intended to work with the Loco Positioning System in TWR TOA
mode. It aims at documenting how to set the Crazyflie in position control mode
and how to send setpoints.
```

From what I understand, this example can also be used with the TDOA mode right?

**Comment by krichardsson** [comment]
Hi,
yes no problem, it should work with any absolution positioning system.

**Comment by knmcguire** [comment]
This belongs in discussions so I'll transfer it.
